libparsec/crates/types/src/certif.rs

```diff
@@ -108,6 +108,13 @@ macro_rules! impl_unsecure_load {
                 pub fn hint(&self) -> String {
                     format!("{:?}", self.unsecure)
                 }
+
+                /// Convert to it's verified variant by check its signature
```

Comment:
```suggestion
                /// Convert to its verified variant by check its signature
```

To be honest I'm not convinced by what added information this docstring brings (since the signature makes it already clear that a `UnsecureFoo` is turned into a `Foo` by doing a `verify_signature` ...), however I don't mind keeping it if you think otherwise :+1: